src/model/database/db_build.sql

```diff
@@ -0,0 +1,33 @@
+BEGIN; 
+
+DROP TABLE IF EXISTS users , reservations , rooms CASCADE ;
+
+CREATE TABLE users(
+    id serial primary key ,
+    first_name VARCHAR(20) NOT NULL,
+    last_name VARCHAR(20) NOT NULL,
+    email_address VARCHAR(50) UNIQUE NOT NULL  ,
+    phone_num INTEGER UNIQUE NOT NULL,
```

Comment:
the `INTEGER` type will remove preceding `0`s,
for example: `0599946544` will be stored as `599946544`
Is that ok?

Reply:
i think we should store it as varchar ??

Reply:
yes I think it's the right choice

I found this ![link](https://stackoverflow.com/questions/24353778/which-is-best-data-type-for-phone-number-in-mysql-and-what-should-java-type-mapp)
when you have time take a look